src/Credentials/UserRefreshCredentials.php

```diff
@@ -42,17 +46,21 @@ class UserRefreshCredentials extends CredentialsLoader
    *
    * @param array $jsonKey JSON credentials.
    *
-   * @param string $jsonKeyPath the path to a file containing JSON credentials.  If
-   *   jsonKey is set, it is ignored.
+   * @param string|array $jsonKey JSON credential file path or JSON credentials
+   *   as an associative array
    */
   public function __construct(
     $scope,
```

Comment:
You're right. It's just for the cache key. The caching implementation in this library could use some consideration, since cache keys would be the same for multiple users with the same scope.